vcpkg/commands/new.md

```diff
@@ -0,0 +1,146 @@
+---
+title: vcpkg new
+description: Command line reference for the vcpkg new command. Creates a new manifest in the current directory.
+ms.author: bion
+ms.date: 07/05/2023
+ms.prod: vcpkg
+---
+# vcpkg new
+
+## Synopsis
+
+```console
+vcpkg new --application
+```
+
+```console
+vcpkg new --name hello --version 1.0
+```
+
+Creates a manifest file ([`vcpkg.json`](../reference/vcpkg-json.md)) and a default configuration file ([`vcpkg-configuration.json`](../reference/vcpkg-configuration-json.md)) in the current working directory.
+Creates a manifest file (`vcpkg.json`) and default configuration file (`vcpkg-configuration.json`)
+ready to use in your project.

```

Comment:
```suggestion
```

Reply:
Hmmm I think this got confused by applying multiple suggestions. Will fix!